lua/wikis/rainbowsix/MatchGroup/Legacy/Default.lua

```diff
@@ -26,6 +26,7 @@ function MatchGroupLegacyDefault:getMap()
 		t1ban2 = 'map$1$t1ban2',
 		t2ban1 = 'map$1$t2ban1',
 		t2ban2 = 'map$1$t2ban2',
+		bantype = 'map$1$bantype',
```

Comment:
All legacy matches will be siege as bantype. Hardcode that instead of needing to add another parameter to the legacy wikicode